File: SIZCapi/Models/PozycjaMenu.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace SIZCapi.Models
{
    public class PozycjaMenu
    {
        public int PozycjaMenuID { get; set; }

        [Column(TypeName = "nvarchar(50)")]
        public string NazwaPozycja { get; set; } 

        [Column(TypeName = "money")]
        public decimal Cena { get; set; }

        public string Opis { get; set; }

        public string ObrazekUrl { get; set; }

        //relacja Zamowienie
        public virtual ICollection<Zamowienie> Zamowienie { get; set; }

        //relacja Skladnik
        public virtual ICollection<Skladnik> Skladnik { get; set; }
    }
}

File: SIZCspa/src/app/_models/pozycjaMenu.ts
import { DlaPozycjaMenuSkladnik } from './dlaPozycjaMenuSkladnik';

export interface PozycjaMenu {
    pozycjaMenuID: number;
    nazwaPozycja: string;
    cena: number;
    opis: string;
    obrazekUrl: string;
    skladnik?: DlaPozycjaMenuSkladnik[];
}
